spec/pagerduty_spec.rb

```diff
@@ -19,7 +18,7 @@
           expect(transport).to have_received(:send_payload).with(
             service_key: "a-test-service-key",
             event_type:  "trigger",
-            description: "a-test-description",
+            description: "a-test-description"
```

Comment:
Yes it does
https://github.com/bbatsov/ruby-style-guide#no-trailing-params-comma